os/fs/smartfs/README.txt

```diff
@@ -1,17 +1,20 @@
-SMARTFS README
-^^^^^^^^^^^^^^
+jSmartFs README
```

Comment:
PR obsolete, newer version of the changes have been committed and merged
Please refer to #5770